src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -220,6 +228,21 @@ protected function getRouteData(Route $route): array
         return $data;
     }
 
+    protected function sortParameters(ParameterBag $parameters): array
+    {
+        $sortedParameters = parent::sortParameters($parameters);
+
+        foreach ($parameters->allDeprecated() as $parameter => $deprecation) {
+            $value = $sortedParameters[$parameter];
+            $sortedParameters[$parameter] = [
+                'value' => $value,
+                'deprecated' => sprintf('Deprecated since %s %s: %s', $deprecation[0], $deprecation[1], sprintf(...\array_slice($deprecation, 2))),
```

Comment:
```suggestion
                'deprecated' => sprintf('Since %s %s: %s', $deprecation[0], $deprecation[1], sprintf(...\array_slice($deprecation, 2))),
```